src/main/java/core/basesyntax/reportservices/ReportToFileImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.reportservices;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+public class ReportToFileImpl implements ReportToFile {
+
+    @Override
+    public void writeReportToFile(List<String> report, String fileName) {
```

Comment:
According to the checklist, avoid hardcoded values. The method should not assume the file location. It's better to receive the file path as a parameter to the method.